service-jehlomat/src/main/kotlin/services/DatabaseService.kt

```diff
@@ -341,6 +349,9 @@ class DatabaseService(
     fun updateSyringe(syringe: Syringe) {
         databaseInstance.update(SyringeTable) {
             updateSyringeRecord(this, it, syringe)
+            where {
+                it.id eq syringe.id
```

Comment:
I would just add a test that verifies this functionality because newcomers would easier understand the "trap" in the Ktorm framework from a test.

Reply:
Test has been added